### PR TITLE
[MRG+1] FIX: Plotting functions to use matplotlib colormaps 'Paired', 'Set1' with colorbar=True

### LINKED ISSUE #1294: I can't make the new colormaps (magma, viridis...) plot the colorbar.
Hi,

I can't make the new colormaps (plt.cm.magma, viridis...) plot the colorbar.
The brain plot is done, but I get an exception when plotting the colorbar.
Has anyone had this problem before?
Thanks.

Cheers,
Alex

### PR DESCRIPTION
**Pull request by KamalakerDadi**
Attempts to Fix #1294. 

Can I have reviews please ?

### CODE REVIEW

**Comment by GaelVaroquaux** [comment]
That's probably the wrong fix: we just need to find what is the way to initiate a segmented colormap from a list in the latest matplotlib.

**Comment by KamalakerDadi** [comment]
Otherwise, we could make a small function to take string as the input argument. Then, load the string to
Linear Segmented Colormap ?

File: nilearn/plotting/tests/test_img_plotting.py [hunk 1]
```diff
@@ -896,3 +896,15 @@ def test_plot_stat_map_with_nans():
     plot_epi(img)
     plot_stat_map(img)
     plot_glass_brain(img)
+
+
+def test_plotting_functions_with_cmaps():
+    img = load_mni152_template()
+    # some cmaps such as 'viridis' (the new default in 2.0), 'magma', 'plasma',
+    # and 'inferno' are not supported for older matplotlib version from < 1.5
+    cmaps = ['Paired', 'Set1', 'Set2', 'Set3']
```
Comment (GaelVaroquaux) [inline nilearn/plotting/tests/test_img_plotting.py]: Can you also use viridis if the maptlotlib version is >= 2.0

Comment (KamalakerDadi) [inline nilearn/plotting/tests/test_img_plotting.py]: Sure.

**Comment by GaelVaroquaux** [comment]
I am suggesting what I think is a better fix: https://github.com/KamalakerDadi/nilearn/pull/7

File: nilearn/plotting/tests/test_img_plotting.py (hunk 1 above)
Comment (KamalakerDadi) [inline nilearn/plotting/tests/test_img_plotting.py]: Comment fixed. Done

**Comment by GaelVaroquaux** [comment]
LGTM. Thanks

@alexsavio, do you want to check that it does fix #1294 ?

**Comment by GaelVaroquaux** [comment]
OK, merging this.

**Pull request merged**